src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.GetRequiredWorkloads.CrossTargeting.targets

```diff
@@ -0,0 +1,33 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.GetRequiredWorkloads.CrossTargeting.targets
```

Comment:
see Daniel's comment above

Reply:
How about simply `Workloads`?

Reply:
How about simply `Workloads` instead of `GetRequiredWorkloads`? It's expected that we'll be adding logic in the future. We can't rename the file then, so might as well give a generic name, IMHO!

Also, the file `Microsoft.NET.Sdk.ImportWorkloadsCrossTarget.targets` still remains after rename, is it intentional?

Reply:
We can rename the file in the future if we want to, though we don't often do so.

I do like the suggested name of `Microsoft.NET.Sdk.Workloads.CrossTargeting.targets`.